addon/components/lt-body.js

```diff
@@ -175,6 +178,54 @@ export default Component.extend({
   useVirtualScrollbar: false,
 
   /**
+   * Set this property to scroll to a specific px offset.
+   *
+   * This only works when `useVirtualScrollbar` is `true`, i.e. when you are
+   * using fixed headers / footers.
+   *
+   * @property scrollTo
+   * @type {Number}
+   * @default null
+   */
+  scrollTo: null,
+
+  /**
+   * Set this property to a `Row` to scroll that `Row` into view.
+   *
+   * This only works when `useVirtualScrollbar` is `true`, i.e. when you are
+   * using fixed headers / footers.
+   *
+   * @property scrollToRow
+   * @type {Row}
+   * @default null
+   */
+  scrollToRow: null,
+
+  /**
+   * @property targetScrollOffset
+   * @type {Number}
+   * @default 0
+   * @private
+   */
+  targetScrollOffset: 0,
```

Comment:
Due to this property triggering an initial scroll, this PR is blocked until https://github.com/alphasights/ember-scrollable/pull/26 gets published.